Content.Shared/_BRatbite/EmpWearable/SharedItemSummoningWearableSystem.cs

```diff
@@ -0,0 +1,35 @@
+using Content.Shared.Actions;
+
+namespace Content.Shared._BRatbite.EmpGlove;
+
+/// <summary>
+/// This handles...
+/// </summary>
+public abstract  class SharedItemSummoningWearableSystem : EntitySystem
+{
+    [Dependency] protected readonly SharedActionsSystem _actionsSystem = default!;
+    [Dependency] private readonly ActionContainerSystem _actionContainer = default!;
+    /// <inheritdoc/>
+    public override void Initialize()
+    {
+        SubscribeLocalEvent<ItemSummoningWearableComponent, GetItemActionsEvent>(OnGetActions);
+        SubscribeLocalEvent<ItemSummoningWearableComponent, MapInitEvent>(OnMapInit);
+    }
+
+    private void OnMapInit(EntityUid uid, ItemSummoningWearableComponent component, MapInitEvent args)
```

Comment:
```suggestion
    private void OnMapInit(Entity<ItemSummoningWearableComponent> ent, ref MapInitEvent args)
```
You VILL use entity<t>